src/chains/ethereum/transaction/src/eip2930-access-list-transaction.ts

```diff
@@ -0,0 +1,229 @@
+import {
+  Data,
+  Quantity,
+  keccak,
+  BUFFER_EMPTY,
+  BUFFER_32_ZERO,
+  RPCQUANTITY_EMPTY
+} from "@ganache/utils";
+import { Address } from "@ganache/ethereum-address";
+import type Common from "@ethereumjs/common";
+import { BN, ecsign } from "ethereumjs-util";
+import { TypedRpcTransaction } from "./rpc-transaction";
+import { encodeRange, digest } from "@ganache/rlp";
+import { RuntimeTransaction } from "./runtime-transaction";
+import {
+  EIP2930AccessListDatabasePayload,
+  EIP2930AccessListDatabaseTx,
+  TypedDatabaseTransaction
+} from "./raw";
+import { AccessList, AccessListBuffer } from "@ethereumjs/tx";
+import { AccessLists } from "./access-lists";
+import { computeInstrinsicsAccessListTx } from "./signing";
+
+const CAPABILITIES: any[] = [2718, 2930];
+export class EIP2930AccessListTransaction extends RuntimeTransaction {
+  public chainId: Quantity;
+  public accessList: AccessListBuffer;
+  public accessListJSON: AccessList;
+  public gasPrice: Quantity;
+  public type: Quantity = Quantity.from("0x1");
+
+  public constructor(
+    data: EIP2930AccessListDatabasePayload | TypedRpcTransaction,
+    common: Common
+  ) {
+    super(data, common);
+    if (Array.isArray(data)) {
+      this.chainId = Quantity.from(data[0]);
+      this.nonce = Quantity.from(data[1], true);
```

Comment:
I think this should be:

```suggestion
      this.nonce = Quantity.from(data[1]);
```